drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -68,6 +68,10 @@ struct aci_set_ble_addr {
 	uint8_t value[6];
 } __packed;
 
+#if defined(CONFIG_BT_HCI_SETUP)
+static int bt_ipm_setup(const struct device *dev, const struct bt_hci_setup_params *params);
```

Comment:
It doesn't look like you need this forward declaration. The calls to the function seem to be after the function definition anyway.

Reply:
Yes, indeed it's useless